### DISCUSSION TITLE
how to introspect prisma1 cloud

### DISCUSSION THREAD

**Discussion by rosnk** [prisma/prisma#14902] [Q&A] [answered]
how do you introspect prisma1 cloud, since we need to mention all the database details(hostname, username, password), 
but we only have cloud url link something like: https://us1.prisma.sh/.../dev in prisma.yml file

I need this because, previous developer did not bother to push the updated prisma datamodel file to git repo, so, i need to introspect from the cloud, also i do not have credential of attached heroku account.

**Comment by ludralph** [comment]
Hi @rosnk 👋 

To perform introspection using prisma1, the command is 

```
prisma1 introspect
```
See the [documentation](https://v1.prisma.io/docs/1.34/prisma-cli-and-configuration/cli-command-reference/prisma-introspect-xcv3/) for more information.

**Comment by andrewicarlson** [comment, marked as answer]
@rosnk as we discussed over email the connection string to databases hosted by Prisma 1 Cloud is not available directly. If you need to introspect locally you can either use the `prisma1 export` CLI command or email prisma1cloud@prisma.io for another dump of your database.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
